changes/issue-5408-transparency-url

```diff
@@ -0,0 +1,4 @@
+- Add `fleet_desktop.transparency_url` to `app_config_json`
+- Set default `transparency_url="https://fleetdm.com/transparency`
+- Enable Fleet Premium licensees to set custom `transparency_url` via REST API and `fleetctl apply`
```

Comment:
typo licenses

Reply:
This was intentional ("licensee" = the holder of a license :female-judge:)

Reply:
🙈 😄